### PR TITLE
Redirect video content to video.ft.com

### PR DESCRIPTION
**Pull request by i-like-robots**
CC https://github.com/Financial-Times/next-es-interface/pull/448

### CODE REVIEW

**Comment by i-like-robots** [comment]
Needs https://github.com/Financial-Times/next-article/pull/1499

File: server/controllers/negotiation.js [hunk 1]
```diff
@@ -60,6 +60,10 @@ module.exports = function negotiationController(req, res, next) {
 				return res.redirect(302, `${webUrl}${webUrl.includes('?') ? '&' : '?'}ft_site=falcon&desktop=true`);
 			}
 
+			if (webUrl.includes('video.ft.com')) {
+				return res.redirect(302, webUrl + 'ft_site=falcon&desktop=true');
```
Comment (matthew-andrews) [inline server/controllers/negotiation.js]: need a `?`?

File: test/server/controllers/negotiation.test.js [hunk 2]
```diff
@@ -237,6 +237,26 @@ describe('Negotiation Controller', function() {
...
+							webUrl: 'http://video.ft.com/5030468875001"'
+						}
+					}]
+				});
+
+			return createInstance({
+				params: { id: 'uuid' }
+			})
+				.then(() => {
+					expect(response.statusCode).to.equal(302);
```
Comment (matthew-andrews) [inline test/server/controllers/negotiation.test.js]: probably wise to add `expect(response.headers.Location).to.equal(/video\.ft\.com\/5030468875001?ft_site=falcon/)`

Comment (i-like-robots) [inline test/server/controllers/negotiation.test.js]: Have added expectations to all redirect specs

**Comment by matthew-andrews** [comment]
✌️

**Comment by matthew-andrews** [comment]
you might need to clear cache + rebuild

**Comment by i-like-robots** [comment]
Will wait for article builds to go green again shoudl any videos be prematurely exposed.

**Pull request merged**